api_v3/lib/types/EntryServerNode/base/KalturaEntryServerNode.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ * @abstract
+ */
+abstract class KalturaEntryServerNode extends KalturaObject implements IRelatedFilterable, IApiObjectFactory
```

Comment:
The is a bit of mismatch in the readOnly and insertOnly for the below attributes.
Either make all of them readOnly or make attributes such as: entryId & serverNodeId & status & serverType as inserOnly